ipn/ipnlocal/local.go

```diff
@@ -168,6 +168,7 @@ type LocalBackend struct {
 	logFlushFunc          func()           // or nil if SetLogFlusher wasn't called
 	em                    *expiryManager   // non-nil
 	sshAtomicBool         atomic.Bool
+	webclientAtomicBool   atomic.Bool
```

Comment:
Thoughts on just `webAtomicBool`?

Reply:
And making the pref just `RunWeb`?

Reply:
I worry that just calling it "web" is too general, as I could easily imagine other web-related things in the future unrelated to the web client.  Though I guess we do just use "web" in several other places... maybe that's an argument for renaming those for consistency?  wdyt?

Reply:
Discussed in sync this morning, and decided to go with the more explicit "webclient" naming.